Frontend/src/views/students/Students.jsx

```
@@ -13,35 +19,193 @@ import Table from 'react-bootstrap/Table';
  *  program: string
  * }]
  */
+
+ const boxStyles = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: '28%',
+  maxHeight: '100%',
+  bgcolor: 'background.paper',
+  boxShadow: 100,
+  padding: '20px',
+  paddingBottom: '30px',
+  borderRadius: '8px'
+};
+
+
 const Students = () => {
+  
+  const [students, setStudents] = useState([]);
+  const [openDeleteModal, setOpenDeleteModal] = useState(false);
+  const [openEditModal, setOpenEditModal] = useState(false);
+  const [selectedStudentId, setSelectedStudentId] = useState(1); 
+  const [selectedStudent, setSelectedStudent] = useState({});
+
+  const getStudents = async () => {
+    try {
+      const response = await fetch(
+        'http://localhost:3100/student'
+      );
+      const jsonData = await response.json();
+      const filteredData = jsonData.filter((student) => student.userId === Number(getCurrentUserId()));
+      setStudents(filteredData);
+    } catch (error) {
+      console.log(error.message);
+    }
+  };
+
+  const deleteStudent = async (studentId) => {
+    try {
+      await fetch(`http://localhost:3100/student/${studentId}`, {
+        method: 'DELETE',
+      });
+    } catch (error) {
+      console.log(error.message);
+    }
```

Good use of `DELETE`.

However, the error was not handled properly.